addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1174,6 +1160,67 @@ function _applyColspan(element, colspan, tableWidth) {
     element.style.setProperty('max-width', width + 'px');
     element.classList.toggle('o_converted_col', true);
 }
+/**
+ * Take an element with a background image and return a string containing the
+ * VML code to display the same image properly in Outlook, with its contents
+ * inside.
+ * TODO: We assume:
+ *      - background-size: cover,
+ *      - background-repeat: no-repeat,
+ *      - size 100%
+ *      - content is centered x/y
+ * These assumptions may not hold for cover snippet and other uses. This needs
+ * to account for that.
+ * TODO: centering span probably not needed with `v-text-anchor:middle` present.
+ *
+ * @param {Element} backgroundImage
+ * @returns {string}
+ */
+function _backgroundImageToVml(backgroundImage) {
+    // Create the outer structure.
+    const clone = backgroundImage.cloneNode(true);
+    const div = document.createElement('div');
+    div.replaceChildren(...clone.childNodes);
+    [['fontSize', 0], ['height', '100%'], ['width', '100%']].forEach(([k, v]) => div.style[k] = v);
+    const vmlContent = document.createElement('div');
+    vmlContent.append(div);
+
+    // Preserve important inherited properties without ancestor context.
+    const style = getComputedStyle(backgroundImage);
+    for (const prop of FONT_PROPERTIES_TO_INHERIT) {
+        div.style[prop] = backgroundImage.style[prop] || style[prop];
+    }
+    [...div.children].forEach(child => child.style.setProperty('font-size', child.style.fontSize || style.fontSize));
+
+    // Prepare the top element for hosting the VML image.
+    for (const prop of ['background', 'background-image', 'background-repeat', 'background-size']) {
+        clone.style.removeProperty(prop);
+    }
+    clone.style.padding = 0;
+    clone.className = clone.className.replace(/p[bt]\d+/g, ''); // Remove padding classes.
+    const url = backgroundImage.style.backgroundImage.match(/url\("?(.+?)"?\)/)[1];
+    clone.setAttribute('background', url);
+    clone.setAttribute('valign', 'middle');
+
+    // Create the VML structure, with the content of the original element inside.
+    const [width, height] = [_getWidth(backgroundImage), _getHeight(backgroundImage)];
+    const vml = `<v:image xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" ` +
+        `style="border: 0; display: inline-block; width: ${width}px; height: ${height}px;" src="${url}"/>
+    <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" ` +
+        `style="border: 0; display: inline-block; position: absolute; width:${width}px; height:${height}px; v-text-anchor:middle;">
+        <v:fill opacity="0%" color="#000000"/>
+        <v:textbox inset="0,0,0,0">
+            <table border="0" cellpadding="0" cellspacing="0">
+                <tr>
+                    <td width="${width}" align="center" style="text-align: center;">${vmlContent.outerHTML}</td>
+                </tr>
+            </table>
+        </v:textbox>
+    </v:rect>`;
```

Comment:
Is there a particular reason why you don't use a template string here ?

Reply:
Sorry I was confused by the `+` above. Why use *two* string templates then ? ^^'

Reply:
It was just to avoid having a super long line or an element that looks like this:
```xml
<my-elememt my-attribute="my-value"
        style="ugly: true;">
```